tf/examples/Bonsai/bonsai_example.py

```diff
@@ -10,101 +11,113 @@
 
-    dataDir = args.data_dir
+	data_dir = args.data_dir
 
-    outFile = args.output_file
+	outFile = args.output_file
+	print ("outFile : ",outFile)
 
-    (dataDimension, numClasses,
-        Xtrain, Ytrain, Xtest, Ytest) = helpermethods.preProcessData(dataDir)
+	(dataDimension, numClasses,
+	    Xtrain, Ytrain, Xtest, Ytest) = helpermethods.preProcessData(data_dir,isRegression)
 
-    sparZ = args.sZ
+	sparZ = args.sZ
 
-    if numClasses > 2:
-        sparW = 0.2
-        sparV = 0.2
-        sparT = 0.2
-    else:
-        sparW = 1
-        sparV = 1
-        sparT = 1
+	if numClasses > 2:
+	    sparW = 0.2
+	    sparV = 0.2
+	    sparT = 0.2
+	else:
+	    sparW = 1
+	    sparV = 1
+	    sparT = 1
 
-    if args.sW is not None:
-        sparW = args.sW
-    if args.sV is not None:
-        sparV = args.sV
-    if args.sT is not None:
-        sparT = args.sT
+	if args.sW is not None:
+	    sparW = args.sW
+	if args.sV is not None:
+	    sparV = args.sV
+	if args.sT is not None:
+	    sparT = args.sT
 
-    if args.batch_size is None:
-        batchSize = np.maximum(100, int(np.ceil(np.sqrt(Ytrain.shape[0]))))
-    else:
-        batchSize = args.batch_size
+	if args.batch_size is None:
+	    batch_size = np.maximum(100, int(np.ceil(np.sqrt(Ytrain.shape[0]))))
+	else:
+	    batch_size = args.batchSize
 
-    useMCHLoss = True
+	useMCHLoss = True
 
-    if numClasses == 2:
-        numClasses = 1
+	if numClasses == 2:
+	    numClasses = 1
 
-    X = tf.placeholder("float32", [None, dataDimension])
-    Y = tf.placeholder("float32", [None, numClasses])
+	X = tf.placeholder("float32", [None, dataDimension])
+	Y = tf.placeholder("float32", [None, numClasses])
 
-    currDir = helpermethods.createTimeStampDir(dataDir)
+	#currDir = bonsaipreprocess.createDir(data_dir)
+	currDir = data_dir
 
-    helpermethods.dumpCommand(sys.argv, currDir)
+	print ("Num of classes : ",numClasses)
+	# numClasses = 1 for binary case
+	bonsaiObj = Bonsai(numClasses, dataDimension,
+		           projectionDimension, depth, sigma,isRegression = isRegression)
 
-    # numClasses = 1 for binary case
-    bonsaiObj = Bonsai(numClasses, dataDimension,
-                       projectionDimension, depth, sigma)
+	bonsaiTrainer = BonsaiTrainer(bonsaiObj,
+		                      regW, regT, regV, regZ,
+		                      sparW, sparT, sparV, sparZ,
+		                      learningRate, X, Y, useMCHLoss, outFile)
 
-    bonsaiTrainer = BonsaiTrainer(bonsaiObj,
-                                  regW, regT, regV, regZ,
-                                  sparW, sparT, sparV, sparZ,
-                                  learningRate, X, Y, useMCHLoss, outFile)
+	sess = tf.InteractiveSession()
+	sess.run(tf.group(tf.initialize_all_variables(),
+		          tf.initialize_variables(tf.local_variables())))
+	saver = tf.train.Saver()
 
-    sess = tf.InteractiveSession()
-    sess.run(tf.global_variables_initializer())
+	dict = bonsaiTrainer.train(batch_size, totalEpochs, sess,
```

Comment:
Please align the second line with the braces. PEP8 checker should do this automatically.